ducktape/cluster/finite_subcluster.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.cluster.cluster import Cluster
+
+
+class FiniteSubcluster(Cluster):
+    """This cluster class gives us a mechanism for allocating finite blocks of nodes from another cluster.
+    """
+    def __init__(self, node_list):
```

Comment:
nit: why not just nodes? naming for parameters matters since the caller can always use named parameters